06.10.2025/sortedList_exercise.c

```diff
@@ -0,0 +1,135 @@
+#include "sortedList.h"
+#include <ctype.h>
+#include <stdio.h>
+
+#define PROMPT " > "
+
+static void printHelpInfo(void)
+{
+    printf("You are in interactive mode, enter:\n"
+           "0 - to quit\n"
+           "1 - to add new value to the sorted list\n"
+           "2 - to remove value from the list\n"
+           "3 - to print the list\n"
+           "4 - to print help info\n");
+}
+
+/*
+ * Skip spaces in string and return first non space character.
+ */
+static char skipSpaces(char* buf)
+{
+    for (; isspace(*buf); buf++)
+        ;
```

Comment:
```suggestion
    while (isspace(*buf))
        buf++;
```

Как Вы понимаете, я от Вас с этим не отстану.